clang-tools-extra/clang-tidy/ClangTidy.cpp

```diff
@@ -149,26 +149,12 @@ class ErrorReporter {
                                    Repl.getLength(), Repl.getReplacementText());
             auto &Entry = FileReplacements[R.getFilePath()];
             Replacements &Replacements = Entry.Replaces;
-            llvm::Error Err = Replacements.add(R);
+            llvm::Error Err = Replacements.addOrMerge(R);
             if (Err) {
```

Comment:
nit: I know this was pre-existing, but this is a more idomatic way to use llvm::Error.
```suggestion
            if (llvm::Error Err = Replacements.addOrMerge(R)) {
```